packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -615,6 +615,14 @@ export class ZWaveNode extends Endpoint {
 		return this._deviceConfig?.label;
 	}
 
+	public get deviceDatabaseUrl(): string {
```

Comment:
If manufacturerId, productType or productId is `undefined`, this should also return `undefined` IMO.